README.md

```diff
@@ -111,12 +111,12 @@ To use `LOG_GROUP_OPTIONS`, set the variable to a JSON string with log group nam
 
 ```javascript
 {
-  "API-Gateway-Execution-Logs_abcdef12345/production": {
+  "/API-Gateway-Execution-Logs_abcdef12345/production": {
```

Comment:
Curious, why adding the / here?

Reply:
Value for AWS_LAMBDA_LOG_GROUP begins with a "/" - let me know if these values should still be specified without the "/" - I'll change the code accordingly.